package.json

```diff
@@ -2,23 +2,27 @@
   "name": "cascading-color-systems",
   "version": "0.1.0-beta.11",
   "description": "generate dynamic color palettes with custom properties",
+  "title": "Cascading Color Systems",
   "main": "dist.js",
   "module": "index.js",
-  "homepage": "https://cascading-colors.netlify.com",
-  "repository": "mirisuzanne/cascading-color-system",
+  "homepage": "https://oddbird.net/cascading-colors/",
```

Comment:
@mirisuzanne Hm, maybe we should actually hold off on this particular change until OddLeventy is launched?

Reply:
yeah, probably.